src/mini_file_server/core/handler.clj

```diff
@@ -50,6 +53,16 @@
     (response {:result
                (and (some? new-name)
                     (apply fs/rename (map fs/path-for [old-name new-name])))}))
+  (DELETE "/*" {{filename :*} :params}
+    (let [path (fs/path-for filename)]
+      (log/info (format "Deleting %s: %s" filename path))
+      (response {:result (fs/delete path)}))))
+
+(defroutes app-routes
+  (route/resources "/")
+  (GET "/" [] (index/render))
+  (GET "/list.json" [] (response (fs/files)))
+  (wrap-basic-authentication update-routes authenticated?)
```

Comment:
.. And not wrapping the routes when the variable is not defined?

Reply:
[Make auth optional and use the environment variable MFS_AUTH.](https://github.com/junegunn/mini-file-server/pull/2/commits/3d8761b26fa3cc982ddda01239aeb077801e906d)